### PR TITLE
irssi 1.2.0

### PR DESCRIPTION
**Pull request by dylancarlson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Test failing on high Sierra

**Comment by javian** [comment]
We've had this formula submitted before and no one has been able to crack the issue with the test failure yet =/ Most recent ones were #36980 #35864

File: Formula/irssi.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  url "https://github.com/irssi/irssi/releases/download/1.2.0/irssi-1.2.0.tar.xz"
+  sha256 "1643fca1d8b35e5a5d7b715c9c889e1e9cdb7e578e06487901ea959e6ab3ebe5"
 
   bottle do
     rebuild 1
     sha256 "789041b9cbd74bbae92962e2846baf463ba57b0ae6e94f4dba83f718253223ef" => :mojave
     sha256 "d8a0ff73163c24d2d93bd4b030fa46d22ba7ecb4b27e0bbc604b1877e0298122" => :high_sierra
     sha256 "644e818a4e6718eea2ff0bebf5c8749aa3df552d4a24be6b109e95452b255942" => :sierra
-    sha256 "8716dafbb1a1313d73edb9b9463eb063e59217eb7e32c3676eac05cc420ca42d" => :el_capitan
+    sha256 "8716dafbb112103d73edb9b9463eb063e59217eb7e32c3676eac05cc420ca42d" => :el_capitan
```
Comment (javian) [inline Formula/irssi.rb]: This should not be modified.

File: Formula/irssi.rb [hunk 2]
```diff
@@ -44,7 +44,7 @@ def install
     end
 
     # https://github.com/irssi/irssi/pull/927
-    inreplace "configure", "^DUIfm", "^DUIifm"
+    # inreplace "configure", "^DUIfm", "^DUIifm"
```
Comment (javian) [inline Formula/irssi.rb]: If we remove this inreplace we should take out all the comments related to it as well before merging.

**Review by javian — CHANGES_REQUESTED** [review]

**Pull request closed without merging**